x/evm/keeper/funtoken_state.go

```diff
@@ -10,27 +10,21 @@ import (
 
 	"github.com/NibiruChain/nibiru/eth"
 	"github.com/NibiruChain/nibiru/x/evm"
-	funtoken "github.com/NibiruChain/nibiru/x/evm"
-)
-
-type (
-	funtokenPrimaryKeyType = []byte
```

Comment:
I think the primary key type is easier to follow when reading the collections below than using `[]byte`